src/vbe_info.rs

```diff
@@ -115,50 +216,103 @@ impl fmt::Debug for VBEModeInfo {
     }
 }
 
+/// A VBE colour field.
+///
+/// Descirbes the size and position of some colour capability.
 #[derive(Debug, PartialEq, Copy, Clone)]
 #[repr(C, packed)]
 pub struct VBEField {
+    /// The size, in bits, of the color components of a direct color pixel.
     pub size: u8,
+
+    /// define the bit position within the direct color pixel or YUV pixel of
+    /// the least significant bit of the respective color component.
     pub position: u8
 }
 
 bitflags! {
+    /// The Capabilities field indicates the support of specific features in the graphics environment.
     pub struct VBECapabilities: u32 {
+        /// Can the DAC be switched between 6 and 8 bit modes.
         const SWITCHABLE_DAC = 0x1;     // The DAC can be switched between 6 and 8-bit modes.
+
+        /// Is the controller VGA compatible.
         const NOT_VGA_COMPATIBLE = 0x2; // The controller can be switched into VGA modes.
+
+        /// The operating behaviour of the RAMDAC
         const RAMDAC_FIX = 0x4;         // When writing lots of information to the palette, the blank bit should be used.
```

Comment:
Can you add the note regarding the blank bit to the documentation and remove the redundant comment?

```
        /// The operating behaviour of the RAMDAC.
        ///
        /// When writing lots of information to the RAMDAC, use the blank bit in Function `09h`.
```
Source (page 26): http://www.petesqbsite.com/sections/tutorials/tuts/vbe3.pdf